SETUP.md

```diff
@@ -138,10 +138,18 @@ These steps are for Apple devices running **macOS 14.x**, including those runnin
        ==> Successfully started `redis` (label: homebrew.mxcl.redis)
        ```
    3. macOS will notify you that `redis` has been configured to start automatically upon user login. Confirm this in System Settings --> General --> Login Items --> `redis-server` 
-1. Setup your local **MySql server**
-   1. Start mysql server:
+1. Setup your local **MySQL database server**
+   1. Link MySQL 8
         ```
-        brew services start mysql
+        brew link --force --overwrite mysql@8.0
```

Comment:
What does this accomplish?

Reply:
This ensures that on a macOS development environment that already has MySQL 5.7 installed that MySQL 8 is symlinked into the path so that it's the default/active MySQL installation accessible from the command line, and any pre-existing links/file to the old version are overriden.